commit ead809a5ff377c3c70a5c6ee90433bc30efcdacd
Author: Uwe Schindler <uschindler@apache.org>
Date:   Thu Dec 13 23:04:22 2012 +0000

    Don't revert my revert. I disabled this JVM. But to be able to test this I must be able to run the test? Can you please wait the next time before you commit such shit?
    
    git-svn-id: https://svn.apache.org/repos/asf/lucene/dev/trunk@1421609 13f79535-47bb-0310-9956-ffa450edef68

diff --git a/lucene/test-framework/src/java/org/apache/lucene/util/TestRuleSetupAndRestoreClassEnv.java b/lucene/test-framework/src/java/org/apache/lucene/util/TestRuleSetupAndRestoreClassEnv.java
index fe7cfa6..e87720d 100644
--- a/lucene/test-framework/src/java/org/apache/lucene/util/TestRuleSetupAndRestoreClassEnv.java
+++ b/lucene/test-framework/src/java/org/apache/lucene/util/TestRuleSetupAndRestoreClassEnv.java
@@ -200,9 +200,6 @@ final class TestRuleSetupAndRestoreClassEnv extends AbstractBeforeAfterRule {
           Arrays.toString(avoidCodecs.toArray()));
       throw e;
     }
-    if (Constants.JAVA_VENDOR.startsWith("IBM")) {
-      throw new AssumptionViolatedException("refusing to execute on broken jvm");
-    }
   }
 
   /**
